app/src/main/java/com/vinh/dictionary_1/data/model/Word.java

```diff
@@ -33,6 +34,8 @@
     private String mVEDescription;
     @ColumnInfo(name = "word_ko_dau")
     private String mWordWithoutDiacritic;
+    @Ignore
+    private boolean isBookmarked;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isBookmarked has the same name as a method
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName